ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -84,6 +84,11 @@ def get_columns(bq_schema):
         }
         try:
             if field.policy_tags:
+                col_obj["taxonomy"] = (
+                    PolicyTagManagerClient()
+                    .get_taxonomy(name=field.policy_tags.names[0].split('/policyTags/')[0])
+                    .display_name
+                )
```

Comment:
Let's move these dictionaries into pydantic models as we agreed, wdyt?

Reply:
- .coveragerc file is actually a part of .gitignore but was committed before it was added to the repo. so gitignore didn’t remove it.
- docker-compose-postgres file changes, i have added those changes while i compared with docker-compose, few changes from the mysql docker-compose were missing in the postgres, compared and made the related change.
- col_obj is actually returned from sqlalchemy ( we just override it for bigquery and snowflake and couple of other dbs ) and is generic for all dbs in the form that is being sent as response.

                      col_obj["taxonomy"] = (
                          PolicyTagManagerClient()
                          .get_taxonomy(name=field.policy_tags.names[0].split('/policyTags/')[0])
                          .display_name
                      )
                      col_obj["policy_tags"] = (
                          PolicyTagManagerClient()
                          .get_policy_tag(name=field.policy_tags.names[0])
                          .display_name
                      )

Reply:
sqlalchemy returns with a dictionary most likely for all dbs in get_columns method..